src/test/java/io/paymenthighway/PaymentAPITest.java

```diff
@@ -757,18 +756,16 @@ public void testTokenize() {
       e.printStackTrace();
     }
     // read redirect uri and GET params from it.
-    HttpUriRequest currentReq = (HttpUriRequest) context.getAttribute(
-        HttpCoreContext.HTTP_REQUEST);
+    List<URI> redirectURIs = context.getRedirectLocations();
+    assertEquals(5, redirectURIs.size());
```

Comment:
Sorry for not only bringing this up in the second pass, but is this tying us to the implementation of the hub to closely? Should adding an additional redirect (for example) fail a test here?

Reply:
Yea, this came with a copy-paste from `FormAPIConnectorTest.postCardFormAndReturnLastQueryString`. I agree, little value, will remove from here and original.